corehq/doctypemigrations/continuous_migrate.py

```diff
@@ -25,6 +26,7 @@ def copy_docs(source_db, target_db, doc_ids):
         return
     with IterDB(target_db, new_edits=False) as iter_db:
         for doc in iter_docs(source_db, doc_ids, attachments=True):
+            doc = _insert_attachments(source_db, doc)
```

Comment:
Yeah it's legit confusing. You're right that I could remove `attachments=True` from `iter_docs`. Overall, what we're seeing here is a discrepancy between CouchDB and Cloudant. Cloudant doesn't respect `attachments=true` in views or `_all_docs`, whereas CouchDB does. There's a comment to this effect in the other place where we had to do this, I just was lazy here.